packages/table/src/common/loading.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+
+export interface ILoadable {
```

Comment:
export does not match filename. not convinced this needs its own file, either. why not merge with LoadableContent?